### PR TITLE
refactor: Change ObjectSelector to Selector

### PR DESCRIPTION
**Pull request by hoxbro**
See https://github.com/holoviz/param/issues/728

### CODE REVIEW

**Review by ahuang11 — APPROVED** [review]

File: geoviews/plotting/mpl/chart.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ class WindBarbsPlot(ColorbarPlot):
 
     padding = param.ClassSelector(default=0.05, class_=(int, float, tuple))
 
-    convention = param.ObjectSelector(objects=["from", "to"], doc="""
+    convention = param.Selector(objects=["from", "to"], doc="""
```
[the remark this answers is not in the document]
Comment (maximlt) [reply geoviews/plotting/mpl/chart.py:27]: `default` isn't required for `Selector`, it'll pick the first value in `objects`. `ObjectSelector` doesn't do that, its default is just `None` (in a bit of a limbo state as `None` is not part of `objects` unless declared explicitly, so setting later to `None` raises an error).

So be careful when going from `ObjectSelector` to `Selector`:

- `convention = param.ObjectSelector(objects=["from", "to"])` -> `<Obj>.convention is None`
- `convention = param.Selector(objects=["from", "to"])` -> `<Obj>.convention == 'from'`

[the remark this answers is not in the document]
Comment (hoxbro) [reply geoviews/plotting/mpl/chart.py:27]: Thank you. 

As @ ahuang11 implemented this, he should get to decide what should be done here.

[the remark this answers is not in the document]
Comment (maximlt) [reply geoviews/plotting/mpl/chart.py:27]: Yep.

(Somehow I also forgot about `allow_None`. I don't really remember how all these kwargs interact with each other, tbh it's best to experiment in a notebook and see for yourself what really happens)

[the remark this answers is not in the document]
Comment (ahuang11) [reply geoviews/plotting/mpl/chart.py:27]: Technically, regardless if it defaults to `None` or `from`, it only checks if `convention == "to"`
```
        if self.convention == "to":
            radians -= np.pi
```

However, to be explicit, 
```suggestion
    convention = param.Selector(default="from", objects=["from", "to"], doc="""
```

**Pull request merged**